src/Data/ArrayBuffer/Types.purs

```diff
@@ -15,7 +15,7 @@ type ByteOffset = Int
 -- | Length in bytes of a DataView or ArrayBuffer
 type ByteLength = Int
 
-foreign import kind ArrayViewType
+foreign import data ArrayViewType
```

Comment:
Should this just be `data ArrayViewType`? My memory is a bit hazy on this.

Reply:
Ah... Yeah, it should. Sorry about that.

Reply:
Admittedly I don't have a firm grasp of polykinds at the moment, and I can't find a source to back me up, but I thought `foreign import kind X` became just `data X`. Perhaps @natefaubion knows the full explanation.

Reply:
Actually, I don't remember myself...

Reply:
Based on [what I wrote previously](https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/11-Syntax/03-Type-Level-Programming-Syntax/ps-0.14/02-Basic-Syntax/01-Defining-Custom-Kinds.purs#L9-L11), I think this is correct?

Reply:
Yep! I think once we revert the whitespace changes to ci.yml we can merge it.